src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private static int randomNumber = 101;
```

Comment:
this should also be final, and please name this variable according to naming convention